src/Antiforgery/src/Internal/DefaultAntiforgeryTokenSerializer.cs

```diff
@@ -126,50 +187,87 @@ public string Serialize(AntiforgeryToken token)
     {
         ArgumentNullException.ThrowIfNull(token);
 
-        var serializationContext = _pool.Get();
+        var securityTokenBytes = token.SecurityToken!.GetData();
```

Comment:
Looks like `SecurityToken` can never be null, change the property to be non-null?